.github/workflows/workflow-get-latest-deployed-versions-from-github.yml

```
@@ -0,0 +1,51 @@
+        run: echo "version=$(gh variable get INFRA_VERSION_${{ inputs.environment }} --repo ${{ github.repository }} --json value --jq .value || echo '')" >> $GITHUB_OUTPUT
+
+      - name: Get Infra SHA
+        id: get-infra-sha
+        if: steps.get-infra-version.outputs.version != ''
+        run: |
+          SHA=$(git rev-list -n 1 ${{ steps.get-infra-version.outputs.version }})
+          echo "sha=$SHA" >> $GITHUB_OUTPUT
+
+      - name: Get Latest Deployed Apps Version
+        id: get-apps-version
+        run: echo "version=$(gh variable get APPS_VERSION_${{ inputs.environment }} --repo ${{ github.repository }} --json value --jq .value || echo '')" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:145: Double quote to prevent globbing and word splitting

(shellcheck)